v4l2loopback.c

```diff
@@ -664,6 +665,14 @@ static void v4l2loopback_create_sysfs(struct video_device *vdev)
 	dev_err(&vdev->dev, "%s error: %d\n", __func__, res);
 }
 
+/* Event APIs */
+
+#define V4L2_EVENT_PRI_CLIENT_USAGE  V4L2_EVENT_PRIVATE_START
```

Comment:
For those interested in this, this is a private extension for Ubuntu OEM platforms. And by private, I mean its usage is pretty limited and therefore it was not proposed to the upstream. However, if somehow v4l2loopback project find it useful as an official part of the project, its adoption is certainly welcome, except the concern of the pollution to the private event id namespace here. I recommend you to use a different id offset other than `V4L2_EVENT_PRIVATE_START` as a private use of v4l2loopback itself if necessary, or it may step on others toe.

Reply:
Thank you :) and I think this will benefit all Linux users who own a laptop with a MIPI camera. 
I'll rework this portion and find an appropriate event ID for it.